tests/kafkatest/services/mirror_maker.py

```diff
@@ -98,6 +99,7 @@ def __init__(self, context, num_nodes, source, target, whitelist=None, blacklist
         self.blacklist = blacklist
         self.source = source
         self.target = target
+        self.offsets_storage = offsets_storage
```

Comment:
Yes, I just meant that we can just do something like `assert offsets_storage in ['kafka', 'zookeeper']`